configs/analysis_configs/cytotable_convert.yaml

```diff
@@ -1,7 +1,7 @@
 cytotable_convert:
   params:
     dest_datatype: parquet
-    source_datatype: null
+    source_datatype: sqlite
```

Comment:
Double checking: does this mean CytoSnake is tightly coupled to SQLite source data input for CytoTable?